res/raw/webxdc.js

```diff
@@ -15,6 +15,11 @@ window.webxdc = (() => {
 
     setUpdateListener: (cb) => (update_listener = cb),
 
+    async allUpdates: () => {
+      return JSON.parse(InternalJSApi.getStatusUpdates(0));
+    },
```

Comment:
```suggestion
    allUpdates: () => {
      return Promise.resolve(JSON.parse(InternalJSApi.getStatusUpdates(0)));
    },
```
nitpicking, i think this is more efficient for converting.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve

Reply:
i cannot find anything about "efficient for converting" in the link. do mean that "Promise.resolve" has a wider supported?

at the end, however, i would not care, i just thought i've learned in the dev chat that the async-syntax-sugar is better than the old syntax.

Reply:
@Frando recently told me to not use async when just returning a promise from a function, I guessed this could apply here to, but maybe there is no difference as the promise is created in the function in both cases here.